apps/taiga-lumbermill/src/dashboards/register/register.component.html

```diff
@@ -0,0 +1,160 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+    class="card"
+>
+    <header tuiHeader>
+        <h2
+            tuiTitle
+            [style.text-align]="'center'"
+        >
+            Sign up
+        </h2>
+    </header>
+    <form
+        [formGroup]="form"
+        (submit)="onSubmit()"
+    >
+        <tui-input
+            formControlName="email"
+            type="email"
+            class="block-input"
+            [readOnly]="submitLoader()"
+        >
+            email
+            <input tuiTextfieldLegacy />
+        </tui-input>
+
+        <tui-error
+            formControlName="email"
+            [error]="[] | tuiFieldError | async"
+            [style.margin-bottom]="'0.8rem'"
+            [style.margin-right]="'auto'"
+        />
+
+        <tui-input-password
+            formControlName="password"
+            type="password"
+            class="block-input"
+            [readOnly]="submitLoader()"
+        >
+            password
+            <input tuiTextfieldLegacy />
+        </tui-input-password>
+        <tui-error
+            formControlName="password"
+            [error]="[] | tuiFieldError | async"
+            [style.margin-bottom]="'0.8rem'"
+            [style.margin-right]="'auto'"
+        />
+        <tui-input-password
+            formControlName="passwordAgain"
+            type="password"
+            class="block-input"
+            [readOnly]="submitLoader()"
+        >
+            confirm password
+            <input tuiTextfieldLegacy />
+        </tui-input-password>
+        <tui-error
+            formControlName="passwordAgain"
+            [error]="[] | tuiFieldError | async"
+            [style.margin-bottom]="'0.8rem'"
+            [style.margin-right]="'auto'"
+        />
+        @if (!form.hasError('unambiguousRole') && (form.touched || form.dirty)) {
+            <p
+                class="error"
+                [style.margin-bottom]="'0.8rem'"
+                [style.margin-top]="'0'"
+            >
+                Passwords should be the same
+            </p>
+        }
+        <div class="under-password">
+            <label tuiLabel>
+                <input
+                    checked
+                    formControlName="rememberMe"
+                    tuiCheckbox
+                    type="checkbox"
+                    [disabled]="submitLoader()"
```

Comment:
remove it, it will disabled with `this.form.disable()`
it causes warning in console (always check devtools console warnings)